modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -16820,9 +16820,9 @@ workflow-index-actions=Workflow Index Actions
 workflow-indexes=Workflow Indexes
 workflow-is-in-use.-remove-its-assignment-to-x=Workflow is in use. Remove its assignment to this asset: {0}.
 workflow-is-in-use.-remove-its-assignment-to-x-and-x=Workflow is in use. Remove its assignment to these assets: {0} and {1}.
-workflow-is-in-use.-remove-its-assignment-to-x-x=Workflow is in use. Remove its assignment to this asset: {0}. {1}
+workflow-is-in-use.-remove-its-assignment-to-x-x=Workflow is in use. Remove its assignment to this asset: {0}.
```

Comment:
@cgmonte I think you should remove the last x at the end of the language key, as follow:
change `workflow-is-in-use.-remove-its-assignment-to-x-x`
to
`workflow-is-in-use.-remove-its-assignment-to-x`

Reply:
@cgmonte @naraandrade, The correct keys already exists, just remove the wrong ones and search for them uses in the portal.